cmd/dep/integration_test.go

```diff
@@ -18,51 +18,52 @@ func TestIntegration(t *testing.T) {
 	test.NeedsExternalNetwork(t)
 	test.NeedsGit(t)
 
-	filepath.Walk("testdata", func(path string, info os.FileInfo, err error) error {
+	filepath.Walk(filepath.Join("testdata", "harness_tests"),
+		func(path string, info os.FileInfo, err error) error {
```

Comment:
I think that everything on one line here is fine - even preferable, because it'll de-dent everything below. Go is lax about the 80-width column thing.

If the width thing really bugs you, assign the func to a var and then call the `filepath.Walk()` below with it 😄